docs/RELEASE_NOTES.md

```diff
@@ -55,13 +55,19 @@
 **Bug Fixes** (1)
 - **Commons** - Fixed library version in User-Agent headers of JVM SDK clients using the library. [#191](https://github.com/commercetools/commercetools-sync-java/issues/191)
 
-**Migration guide** (4)
+**Migration guide** (6)
+- **Product Sync** - Removed `removeOtherVariants` option which is already done by the sync by default. Removal of 
+variants can be prevented through the beforeUpdateCallback. Please check [here](/src/integration-test/java/com/commercetools/sync/integration/ctpprojectsource/products/templates/beforeupdatecallback/KeepOtherVariantsSyncIT.java)
+ for an example of how this can be done. [#26](https://github.com/commercetools/commercetools-sync-java/issues/26)
+- **Commons** - Removed `removeOtherSetEntries`, `removeOtherCollectionEntries` and `removeOtherProperties` options 
+which are already done by the sync by default. Such options can be disbaled through the beforeUpdateCallback. Please 
```

Comment:
`Such options can be disbaled through the beforeUpdateCallback` -> `Mentioned options and even more use cases can now be covered with help of beforeCreateCallback and beforeUpdateCallback`

Reply:
f4b29d9